ClinGen/clincoded
# Connects to #906. VEP attribution icon.

**Steps to test:**
1. Login and use **15333** for selecting the variant.
2. Proceed to the **Basic Info** tab and expect to see the **Ve!P** icon replacing the **VEP** label in the headers of the **RefSeq Transcripts** and **Ensembl Transcripts** tables. Also expect to see the **Ve!P** icon replacing the **VEP** label in the attribution at the bottom of the tab.
3. Proceed to the **Population** tab and expect to see the **Ve!P** icon replacing the **VEP** label in the header of the **1000 Genomes** table. Also expect to see the **Ve!P** icon replacing the **VEP** label in the attribution at the bottom of the tab.
